src/main/java/org/example/models/JobRoleResponse.java

```diff
@@ -0,0 +1,74 @@
+package org.example.models;
+
+import java.util.Date;
+
+public class JobRoleResponse {
```

Comment:
Why do we have this?

Reply:
Can you please answer my question and not just resolve the comment

Reply:
i don't understand. If you mean the imports, we are importing List as there are numerous Location enums
and Date is for the closingdate